packages/peregrine/lib/store/reducers/cart.js

```diff
@@ -28,12 +28,35 @@ const reducerMap = {
             cartId: String(payload)
         };
     },
+    [actions.mergeCarts.request]: state => {
+        return {
+            ...state,
+            isLoading: true
+        };
+    },
     [actions.getDetails.request]: state => {
         return {
             ...state,
             isLoading: true
         };
     },
+    [actions.mergeCarts.receive]: (state, { payload, error }) => {
+        if (error) {
+            return {
+                ...state,
+                detailsError: payload,
```

Comment:
Define a new error property in the store for `mergeError` rather than re-using the `getDetails` property.

Reply:
fixed it.